src/app/design/guidelines/user-assistance/index.html

```diff
@@ -29,6 +29,9 @@
 			<p>
 				Empty-state help provides user assistance when content containers are not populated. The content can include a title, body copy, and buttons to take actions or access additional resources. The help content does not appear when content containers are populated.
 			</p>
+			<p>
+				When possible, use a button to open an inline experience in a modal. If you can't use a button or if users should be sent elsewhere, use a link to imply navigation. Buttons suggest modals while links indicate navigation. 
```

Comment:
The placement of this content feels a off to me. I feel like it breaks up the flow of the existing content. I'd make this a the fourth bullet in the unordered list (before the last bullet). Since the second and third bullets pertain to the title and body copy, I think it makes sense to put guidelines for bullets next so that we follow the same order that we use in the first paragraph.

To make that work, I think we'd need to modify your existing text just a bit, and I also think we should update this to reference the button styles that we recommend. I'm thinking something like: "For the button, use the secondary button class to open a modal where users can complete the experience inline. If users cannot complete the experience inline and must navigate elsewhere, use the link button class instead. Secondary buttons indicate modals, while links indicate navigation." ... I'd make "the secondary button class" and "the link button class" both links to https://developer.blackbaud.com/skyux/components/button. ... What do you think?

Reply:
I like that and updated the PR with this.